plugins/modules/aci_l3out_logical_interface_profile_ospf_policy.py

```diff
@@ -0,0 +1,333 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
```

Comment:
can you add details of you as creator in following format

# Copyright: (c) 2022, <name> (<creds>) <email>

Reply:
fixed